test/decompiler/reference/engine/game/video_REF.gc

```diff
@@ -0,0 +1,111 @@
+;;-*-Lisp-*-
+(in-package goal)
+
+;; definition for function set-video-mode
+;; INFO: Return type mismatch int vs none.
+(defun set-video-mode ((arg0 video-parms))
```

Comment:
I think `arg0` here is likely a `symbol` here and not `video-parms`.  It's compared to `'ntsc` and `'pal`.  Both the compiler/decompiler do very little type checking for comparing things for equality, but it doesn't make sense to compare a structure to a symbol.